osc-server/src/test/java/org/osc/core/broker/service/servicefunctionchain/UpdateServiceFunctionChainServiceTest.java

```diff
@@ -60,91 +60,77 @@ public void testDispatch_WithNullRequest_ThrowsNullPointerException() throws Exc
 	@Test
 	public void testDispatch_WithNonEmptyListOfVsIdsUpdateValidationSuccessful_ValidationSucceeds() throws Exception {
 		
+		//Arrange
```

Comment:
picky `//Arrange` -> `// Arrange.` Observe the space and period. Also check other places.